examples/map/compare_rotation_results.py

```diff
@@ -0,0 +1,47 @@
+# # Comparing Results of `map.rotate` Between Library Implementations
+# ----------
+
+import sunpy.data.sample
+import sunpy.map
+import numbers
+import numpy as np
+import cv2
+
+# Rotating a map in Sunpy has a choice between three libraries: `skimage', `scipy`, and `cv2`.
+# One can also create a custom rotation function and drop it into `map.rotate`.
+# Since all these options use different algorithms, we need a basis for comparison:
+# what is an acceptable margin of difference in the final, rotated data product?
+# One option is to use the Symmetric Mean Absolute Percentage Error
```

Comment:
I'm not sure calling this the "error" is right for this application. The rotation algorithms are just different (one is not "correct" compared to the others), and we're measuring differences between them, not errors. Could we rename "error" > "difference" in this section?